Description: fixes a typo that caused crash on vt100 terminal
 also accepts xterm-256color as valid xterm.
 .
 python-xtermcolor (1.2-1) unstable; urgency=low
 .
   * Initial release. (Closes: #695210)
Author: Salvo 'LtWorf' Tomaselli <tiposchi@tiscali.it>
Bug-Debian: http://bugs.debian.org/695210

--- python-xtermcolor-1.2.orig/xtermcolor/__init__.py
+++ python-xtermcolor-1.2/xtermcolor/__init__.py
@@ -29,10 +29,10 @@ def colorize(string, rgb=None, ansi=None
         colorize.init = True
         colorize.is_term = isatty(fd)
         if 'TERM' in environ:
-            if environ['TERM'] == 'xterm':
+            if environ['TERM'].startswith('xterm'):
                 colorize.cmap = XTermColorMap()
             elif environ['TERM'] == 'vt100':
-                colorize.xterm = VT100ColorMap()
+                colorize.cmap = VT100ColorMap()
             else:
                 colorize.is_term = False
         else:
